test/schema_dumper_extension_test.rb

```diff
@@ -15,14 +15,14 @@
 
       after do
         schema_file.unlink
-        init_schema
+        Phenix.burn!
```

Comment:
We don't necessarily need it but it keeps your local db cleaner to have it. Ideally we'd need some RSpec `after(:suite)` handler.